distributor/distributor.go

```diff
@@ -527,7 +543,7 @@ func (d *Distributor) forAllIngesters(f func(cortex.IngesterClient) (interface{}
 			numErrs++
 		}
 	}
-	if numErrs > (d.cfg.ReplicationFactor - d.cfg.MinReadSuccesses) {
+	if numErrs > 1 {
```

Comment:
So I guess this should really be `floor(d.cfg.ReplicationFactor / 2)` to keep the invariant above... which for RF=3 is 1.  Will make it dynamic.